.github/workflows/renovate.yml

```diff
@@ -0,0 +1,16 @@
+name: Renovate
+on:
+  schedule:
+    # Run every 30 minutes
+    - cron: "0/30 * * * *"
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v40.1.5
+        with:
+          configurationFile: renovate.json
+          token: ${{ secrets.RENOVATE_TOKEN }}
```

Comment:
We need a token for this to work, or we can also use the GitHub App approach: https://github.com/renovatebot/github-action?tab=readme-ov-file#example-with-github-app

Reply:
Done ✅